hashtables/ex1/ex1.py

```diff
@@ -12,6 +10,14 @@ def get_indices_of_item_weights(weights, length, limit):
     """
     YOUR CODE HERE
     """
+    # loop through the list
+    for i in range(length):
+        item = weights[i]
+        diff = hash_table_retrieve(ht, limit - item)
+        if diff is None:
+            hash_table_insert(ht, item, i)
+        else:
+            return (i, diff)
```

Comment:
Great job and clean solution! Can you think of a time this might break? `The higher valued index should be placed in the zeroth index and the smaller index should be placed in the first index.`